src/bp/core/config/config-loader.ts

```diff
@@ -129,18 +130,20 @@ export class ConfigProvider {
   }
 
   public async getModulesListConfig() {
-    const enabledByDefault = [
-      'analytics',
-      'basic-skills',
-      'builtin',
-      'channel-web',
-      'nlu',
-      'qna',
-      'extensions',
-      'code-editor',
-      'testing',
-      'examples'
-    ]
+    const enabledByDefault: string[] = yn(process.env.TEST_NLU)
```

Comment:
is there a way of doing this without changing the default value in the code, like can we override the botpress.config file or it would require a server restart ?

Reply:
I don't think so... I found nothing while searching + @allardy would have told me. I can still start botpress, change the config file then restart the server or write the whole `botpress.config.json` file before starting botpress, but this is kind of hackish and will break each time this file is modified.